src/ParameterParser.h

```diff
@@ -403,6 +403,32 @@ namespace snowcrash {
             }
         }
 
+        static size_t matchBracket(const mdp::ByteBuffer& text) {
```

Comment:
Yup, that one. It has more logic for matching brackets, and we can use the output here to get the last bracket pos. So, it should be okay to use it.